github/ci/prow/templates/mirror-proxy-config.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mirror-proxy-config
+data:
+  ca.srl: "0A"
+  ca.crt: |
```

Comment:
I am undecided if it is better to have it together with the rest, or just adding it here.